DEPS

```diff
@@ -799,7 +799,7 @@ deps = {
         'version': 'version:17'
        }
      ],
-     'condition': 'download_android_deps',
+     # Always download the JDK since java is required for running the formatter.
```

Comment:
DBC: This should continue to be controlled by a variable. It can default to true for local dev, but then CI builds that don't need it can still set it to false.